xAPI.md

```diff
@@ -417,6 +419,17 @@ most LRSs were Learning Management Systems (LMSs); however this document uses th
 LRS to be clear that a full LMS is not necessary to implement the xAPI. The xAPI 
 is dependent on an LRS to function.
 
+<a name="def-metadata-consumer" />
+
+__Metadata Consumer__: A person, organization, software program or other thing that seeks to determine the meaning represented
+by an IRI used within this specification and/or retrieves metadata from an IRL used as an IRI within this specification. An LRS may or
+may not be a metadata consumer. 
+
+<a name="def-metadata-provider" />
+
+__Metadata Provider__: A person, organization, software program or other thing that coins IRIs to be used within this specification and/or
+hosts metadata an IRL used as an IRI within this specification. 
```

Comment:
Though thinking about it, I recommend a rephrase: "and/or hosts metadata about an IRI." It doesn't matter where that metadata is hosted.